.github/workflows/unit-test.yml

```diff
@@ -13,26 +13,13 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-    - name: Checkout code
-      uses: actions/checkout@v2
+      - name: Checkout code
+        uses: actions/checkout@v2
 
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: 3.8
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
 
-#    - name: Install dependencies
-#      run: |
-#        python -m pip install --upgrade pip
-#        pip install -r requirements.txt
-
-#    - name: Load environment variables
-#      uses: decathlon/gh-actions-load-env-file@v1
-#      with:
-#        path: .env
-#
-#    - name: Run tests
-#      run: pytest tests/**/*.py
-#      env:
-#        ENV_VAR_1: ${{ env.BUCKET_NAME }}
-#        ENV_VAR_2: ${{ env.OBJECT_KEY }}
+      - name: Run pytest
```

Comment:
will it run via github action ? And anything need to pass via environment variable

Reply:
We are going to run via github action
The yaml is written and will be in the PR with the remaining unit tests